Dockerfile

```diff
@@ -27,8 +27,8 @@ WORKDIR /usr/src/app
 # install maven dependency packages (keep in image)
 COPY pom.xml /usr/src/app
 
-RUN mvn install
 COPY src /usr/src/app/src
 
+RUN mvn install
```

Comment:
Does `nvm install` has to be after copying the source? This way you won't make use of the docker cache layering.

Reply:
mvn builds to .proto-files and needs to be ran after they are copied